vehicles.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS `player_vehicles` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `license` varchar(50) NOT NULL,
```

Comment:
Is it even needed to associate vehicles to licenses? We already associate them to citizenids.

Reply:
I dont think they need to